canonical-web-and-design/jaas-dashboard
# Move App into /components and add snapshot test

## Done

Move App into /components and add snapshot test

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Run `./run test`
- Verify no errors.

## Details

Fixes #29
